### PR TITLE
Features service

### PR DESCRIPTION
**Pull request by indytechcook**
Adding a features Service and updated the submit controllers

cc @tladendo @vietnogi @r-jain

### CODE REVIEW

**Comment by tladendo** [comment]
@indytechcook -- I see a spec, but no feature service?

**Comment by indytechcook** [comment]
@tladendo that's funny.  True TDD :)

It's there now.

File: src/client/app/submit-work/features/scripts/features.controller.js [hunk 1]
```diff
@@ -19,7 +19,7 @@
     function activate() {
       logger.info('Activated Features View');
       vm.work = SubmitWorkService.getCurrent();
+      vm.work.features = FeatureService.getFeatures();
```
Comment (tladendo) [inline src/client/app/submit-work/features/scripts/features.controller.js]: Doesn't this return a promise?

Comment (indytechcook) [inline src/client/app/submit-work/features/scripts/features.controller.js]: Better?

**Comment by indytechcook** [comment]
@tladendo take another look.

**Comment by tladendo** [comment]
lgtm now

**Pull request merged**